src/main/java/com/yahoo/bullet/bql/extractor/QueryExtractor.java

```diff
@@ -153,6 +161,9 @@ protected Void visitQuerySpecification(QuerySpecification node, Void context) th
                 case SELECT_FIELDS:
                 case SELECT_ALL:
                     extractRaw();
+                    node.getOrderBy().ifPresent(value -> {
```

Comment:
Why do we only add this for SELECT_FIELDS and SELECT_ALL and not GROUP etc?

Reply:
Also can just move this to extractRaw

Reply:
As before, not really sure where ORDER BY can go. Is it everywhere (including TopKFunction) except TopK?